src/OMSimulatorLib/SystemTLM.h

```diff
@@ -57,6 +58,10 @@ namespace oms3
     SystemTLM& operator=(SystemTLM const& copy); ///< not implemented
 
   private:
+    void* model;
+    std::string address;
+    int managerPort;
```

Comment:
This has to be imported/exported from/to SSP, right?

Reply:
I added import/export to SSD.